### DISCUSSION TITLE
collaboration opportunity

### DISCUSSION THREAD

**Discussion by HeiziND** [CrystalWindSnake/ex4nicegui#223] [General]
Hi @CrystalWindSnake , 
I'm of the founder of a china-based business and we are about to start (re)building our internal CRUD and ML toolset using nicegui. Your profile is very much of interest, and I would like to discuss potential collaboration. 
is it possible to catch up on wechat or phone?
我也会说中文，但说得不太流利 ：）

**Comment by CrystalWindSnake** [comment]
@HeiziND Sure, let's follow each other and exchange contact details via direct message.

**Reply by HeiziND** [reply]
Thanks, I followed you but still do not see an option for direct messaging.
Here's my wechat QR code:

[REDACTED]
